README.md

```diff
@@ -9,10 +9,11 @@ MVP at https://www.paironauts.org
 To start the Paironauts Phoenix server:
 
   * Install dependencies with `mix deps.get`
+  * Create Ecto migrations directory using...
+    * (Mac/Linux) using `mkdir -p priv/repo/migrations`
```

Comment:
Can't we just commit an empty migrations directory and spare the space in the README and spare our devs this process?

Reply:
I just did this because one can't commit an empty directory (I think).

Reply:
I'd prefer to do this by adding a .keep to that directory, to be removed later. It's a marginally less involved change and later cleanup and it removes a setup step for contributors